hail/src/main/scala/is/hail/backend/spark/LowerTableIR.scala

```diff
@@ -54,133 +22,45 @@ case class SparkStage(
   partitioner: RVDPartitioner,
   rdds: Map[String, RDD[RegionValue]],
   contextType: Type,
-  context: Array[Any],
+  context: IR,
```

Comment:
Especially now that the type gives us no hints what this is, I think it would be clearer to call it "contexts".